contracts/TrueReward/AssuredFinancialOpportunity.sol

```diff
@@ -187,16 +188,14 @@ contract AssuredFinancialOpportunity is FinancialOpportunity, AssuredFinancialOp
     function setRewardBasis(uint32 newBasis) external onlyOwner {
         minTokenValue = _tokenValue();
 
-        adjustmentFactor = adjustmentFactor
-            .mul(_calculateTokenValue(rewardBasis))
-            .div(_calculateTokenValue(newBasis));
+        adjustmentFactor = adjustmentFactor.mul(_calculateTokenValue(rewardBasis)).div(_calculateTokenValue(newBasis));
```

Comment:
Is there a benefit to making this one line? might be easier to read how it was before. Not a major issue though.

Reply:
Linter did it.